boxoffice/templates/invoice.html

```diff
@@ -0,0 +1,209 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset='UTF-8'>  
+  <title>Invoice</title>  
+  <link rel="stylesheet" href="{{ url_for('static', filename='css/font-awesome.min.css') }}">
+  <link rel="stylesheet" type="text/css" href="//fonts.googleapis.com/css?family=Source+Sans+Pro:400italic,600italic,400,600" />
+  <style type="text/css">
+    * { 
+      margin: 0; 
+      padding: 0; 
+    }
+
+    body { 
+      font-family: "Source Sans Pro",Arial,sans-serif;
+      font-size: 16px;
+      line-height: 1.5;
+      color: #333;
+      background-color: #fff
+    }
+
+    a {
+      text-decoration: none;
+    }
+
+    .clearfix:after, .clearfix:before { 
+      clear: both; 
+      display: table; 
+      content: " "; 
+    }
+
+    .wrapper { 
+      width: 800px; 
+      margin: 20px auto; 
+    }
+
+    .header { 
+      font-size: 40px; 
+      text-align: center; 
+      margin-bottom: 30px; 
+      padding-bottom: 10px; 
+      border-bottom: 1px solid #000;
+    }
+
+    .company-details { 
+      margin-bottom: 40px; 
+    }
+
+    .logo { 
+      float: left;
+    }
+
+    .address { 
+      float: right; 
+    }
+
+    .customer { 
+      margin-bottom: 40px; 
+    }
+
+    table { 
+      border-collapse: collapse; 
+      border: 1px solid #000;
+    }
+
+    table td, table th { 
+      border: 1px solid #000; 
+      padding: 5px; 
+    }
+
+    .center { 
+      text-align: center;
+    }
+
+    .cutomer-details { 
+      float: left; 
+    }
+
+    .details { 
+      float: right; 
+    }
+
+    .item-description {
+      max-width: 200px;
+    }
+
+    .fa {
+      padding-right: 5px;
+    }
+
+    .gray { 
+      background-color: #eee; 
+    }
+
+    .total, .total-amount {
+      font-weight: 600;
+    }
+
+    .items {
+      width: 100%;
+      margin-top: 20px;
+    }
+
+    .terms {
+      margin-top: 20px;
+    }
+
+    .terms-heading {
+      font-size: 15px;
+      font-weight: 600;
+    }
+
+    .terms-content {
+      font-size: 13px;
+    }
+  </style>
+</head>
+
+<body>
+  <div class="wrapper">
+    <h1 class="header">Cash Receipt</h1>
+    
+    <div class="company-details clearfix">  
+      <div class="logo">
+       <img src="https://hasgeek.com/static/img/hg-banner.png"/>
+      </div> 
+
+      <div class="address">
+        <h2>HasGeek Learning Private Limited</h2>
+        <p>141/142, 2nd cross, Pai Layout</p>
+        <p>Hulimavu Gate, Bannerghatta Road</p>
+        <p>Bangalore – 560076</p>
+        <p>India</p>
+        <p>CIN: U74900KA2015PTC083923</p>
+        <p>PAN No: AADCH7324J</p>
```

Comment:
Also, these details should be attributes of the Organization model, not hardcoded.